src/app/events/components/tests-add-entry/tests-add-entry.component.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { buildTest } from 'src/spec-builders';
+import { buildTestModuleMetadata } from 'src/spec-helpers';
+
+import { TestsAddEntryComponent } from './tests-add-entry.component';
+
+describe('TestsAddEntryComponent', () => {
+  let component: TestsAddEntryComponent;
+  let fixture: ComponentFixture<TestsAddEntryComponent>;
+
+  beforeEach(async () => {
```

Comment:
Gut zu wissen, dass dies auch so funktioniert - bisher habe ich das immer in ein `waitForAsync(() => ...)` gewrapped - weil das sonst überall auch so gemacht wurde...